Sources/SwiftSyntaxBuilder/DeclBuildables.swift.gyb

```diff
@@ -0,0 +1,89 @@
+%{
```

Comment:
And then the naming of this file? 
Should it be something like `BuildableExtensions.swift.gyb` ?